javascripts/background_scripts/first_run.js

```diff
@@ -52,14 +52,25 @@ var firstRun = {
     }
 
     // Open the first run page only on new installations.
-    if ( firstRun.getPrivlyVersion() === undefined ) {
-      var page = chrome.extension.getURL("privly-applications/Pages/ChromeFirstRun.html");
-      chrome.tabs.create({
-          url: page,
-          active: true
-      });
+    if (!firstRun.getStoredVersion()) {
+	    var page = chrome.extension.getURL("privly-applications/Pages/ChromeFirstRun.html");
+
+	    chrome.windows.getLastFocused(function(w){
+		      if(w.id){
+			      chrome.tabs.create({
+				      url: page,
+				      active: true,
+				      windowId: w.id
+			      });
+		      }else{
+			      console.log("Cannot get active window");
```

Comment:
You also have a ton of spacing here. Make sure you don't indent more than two spaces per level of nesting.